packages/elephantdsp-roomreverb/PKGBUILD

```
@@ -0,0 +1,85 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
+
+_plugin_name=RoomReverb
+pkgname="elephantdsp-${_plugin_name,,}"
+_plugin_uri="https://www.ElephantDSP.com/plugins/Room_Reverb"
+pkgver=0.6.0
+pkgrel=1
+pkgdesc='A mono and stereo algorithmic room reverb audio effect plugin'
+arch=(x86_64 aarch64)
+url='https://www.elephantdsp.com/products/room-reverb/'
+license=(GPL3)
+depends=(glibc gcc-libs)
+makedepends=(cmake freetype2 'juce>=7' libx11 libxcursor libxinerama libxrandr)
+checkdepends=(lv2lint lv2)
+optdepends=(
+  'clap-host: for CLAP plugin'
+  'lv2-host: for LV2 plugin'
+  'vst3-host: for VST3 plugin'
+)
+groups=(clap-plugins lv2-plugins pro-audio vst-plugins)
+_clap_commit='395ab2bc5abd76f613b7ac3e4292b4e315700749'
+_clap_ext_commit='ff12b486caa173db824691f141b34aad0770d750'
+_clap_helpers_commit='b101259ae06964c76c6806f02ec07847acb7b6dc'
+source=("$pkgname-$pkgver.tar.gz::https://github.com/cvde/RoomReverb/archive/refs/tags/v$pkgver.tar.gz"
+        "clap-juce-extensions-$_clap_ext_commit.zip::https://github.com/free-audio/clap-juce-extensions/archive/$_clap_ext_commit.zip"
+        "clap-$_clap_commit.zip::https://github.com/free-audio/clap/archive/$_clap_commit.zip"
+        "clap-helpers-$_clap_helpers_commit.zip::https://github.com/free-audio/clap-helpers/archive/$_clap_helpers_commit.zip"
```

I'd prefer tar.gz over zip if available:
```suggestion
        "clap-juce-extensions-$_clap_ext_commit.tar.gz::https://github.com/free-audio/clap-juce-extensions/archive/$_clap_ext_commit.tar.gz"
        "clap-$_clap_commit.tar.gz::https://github.com/free-audio/clap/archive/$_clap_commit.tar.gz"
        "clap-helpers-$_clap_helpers_commit.tar.gz::https://github.com/free-audio/clap-helpers/archive/$_clap_helpers_commit.tar.gz"
```
(then update sha256sums)

---

Fixed in d66ec82.